core-bundle/src/EventListener/DoctrineSchema/AdjustSearchUrlLengthListener.php

```diff
@@ -64,14 +66,16 @@ public function __invoke(GenerateSchemaEventArgs $event): void
 
     private function getMaximumIndexSize(Table $table): int
     {
-        $engine = $table->getOption('engine');
+        $engine = $table->hasOption('engine') ? $table->getOption('engine') : 'InnoDB';
```

Comment:
Can we safely assume `InnoDB` as fallback value?

Reply:
It's the same in our existing `MysqlInnodbRowSizeCalculator`. /cc @ausi